assets/src/components/v2/elevator/elevator_closures_list.tsx

```diff
@@ -39,33 +63,55 @@ const ClosureRow = ({ station }: ClosureRowProps) => {
           {closure.elevator_name} ({closure.elevator_id})
         </div>
       ))}
-      <hr className="thin" />
     </div>
   );
 };
 
-const InStationSummary = () => {
+interface InStationSummaryProps {
+  closures: Closure[];
+}
+
+const InStationSummary = ({ closures }: InStationSummaryProps) => {
+  let text;
+  if (closures.length === 0) {
+    text = "All elevators at this station are currently working";
+  } else if (closures.length === 1) {
+    text = (
+      <>
+        <b>This elevator is working.</b> Another elevator at this station is
+        down.
+      </>
+    );
+  } else {
+    text = (
+      <>
+        <b>This elevator is working.</b>{" "}
+        {_.capitalize(numberToWords.toWords(closures.length))} other elevators
```

Comment:
There's a minor design issue we maybe didn't anticipate here, where certain "number words" are long enough to push the following word onto the next line, which cascades into creating a 3-line status message.

<img width="550" alt="Screenshot 2024-12-09 at 10 46 50 AM" src="https://github.com/user-attachments/assets/24124387-8932-486c-8801-a167cada99b5">

My suggestion would be to use the "numeric number" (e.g. "2 other elevators"), or just use some fixed language when more than one other elevator is down, like "Other elevators at this station". Let's see what Betsy thinks.

Reply:
To put a bow on this convo, Betsy agreed with the digit approach: https://mbta.slack.com/archives/C039ARUM48H/p1733760048513209?thread_ts=1733759798.319529&cid=C039ARUM48H